interface/swagger.json

```diff
@@ -1,11 +1,11 @@
 {
     "swagger": "2.0",
     "info": {
-        "version": "1.0.0",
```

Comment:
I would not change this version yet as we have not published an API as of yet.

Thus I suggest sticking to 1.0.0 until this is released (i.e. master branch).

Reply:
done